src/libs/translate.js

```diff
@@ -0,0 +1,57 @@
+import lodashGet from 'lodash.get';
+import Str from 'expensify-common/lib/str';
+import Log from './Log';
+import Config from '../CONFIG';
+import translations from '../languages/translations';
+
+/**
+ * Return translated string for given locale and key
+ *
+ * @param {String} locale
+ * @param {String|Array} key
```

Comment:
`key` is really `word` to translate, right? If that's correct can we rename it to `word` then? Or `phrase`? `key` is a bit vague

Reply:
I wanted to keep it as it is stated in the issue. But `phrase` sounds better.

Reply:
I think `key` is fine... not opposed to phrase though

Reply:
Heh, I prefer `phrase`, would be great if you could change that since we seem to all agree that it sounds better?

Reply:
I changed key with phrase and updated the PR.